### ISSUE TITLE
Collapse and expand buttons in table of contents

### ISSUE THREAD

**Issue by pOuU9w4** [marktext/marktext#3373]
Would it be possible for you to add the collapse and expand buttons in table of contents? It is in the third button on the sidebar.

**Comment by ashishkumarbhati** [comment]
yes this is much needed feature. i find it very difficult to use the app when the document gets too large. there should be a button to expand / collapse all the table of contents. I have multiple docs with over 25000 words each and it is such a pain to navigate.

**Comment by Jocs** [comment]
Closing as a duplicate of #3306, which asks for the same thing: a way to collapse (and expand) all headings in the outline panel at once instead of clicking them one by one.

For reference, the implementation site is the same in both cases — `packages/desktop/src/renderer/src/components/sideBar/toc.vue` tracks only per-node collapse state and exposes no collapse-all/expand-all control.
